core-api/src/routes/access.route.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from "express";
+import { AccessController } from "../controllers/access.controller";
+
+function AuthRouter(): Router {
+  const router = Router();
+
+  router.post("/access", AccessController.create);
+
+  return router;
+}
+
+export default AuthRouter();
```

Comment:
Here as well, and you have to import it in the app.ts file, like the other routes